code/b-parasite/Makefile

```diff
@@ -1,17 +1,40 @@
+# Supported platforms:
+# * E73_2G4M08S1C (nrf2480, default)
+# * E73_2G4M08S1E (nrf2833) 
+PLATFORM ?= E73_2G4M08S1C 

```

Comment:
For some reason I can't get this default value to kick in. If I don't specify a `PLATFORM`, `make` simply bails out and does not compile anything:

```bash
rbaron@mbp.local:~/dev/parasite/b-parasite/code/b-parasite:<feature/nrf52833✔>
% SDK_ROOT=~/dev/nrf52/sdk/nRF5_SDK_17.0.2_d674dde make clean

rm -rf _build
rbaron@mbp.local:~/dev/parasite/b-parasite/code/b-parasite:<feature/nrf52833✔>
% SDK_ROOT=~/dev/nrf52/sdk/nRF5_SDK_17.0.2_d674dde make

make: Nothing to be done for `default'.
rbaron@mbp.local:~/dev/parasite/b-parasite/code/b-parasite:<feature/nrf52833✔>
% SDK_ROOT=~/dev/nrf52/sdk/nRF5_SDK_17.0.2_d674dde PLATFORM=E73_2G4M08S1C make

mkdir _build
cd _build && mkdir nrf52840_xxaa
Assembling file: gcc_startup_nrf52840.S
Compiling file: system_nrf52840.c
Compiling file: nrf_log_backend_rtt.c
Compiling file: nrf_log_backend_serial.c
Compiling file: nrf_log_backend_uart.c
...
```

Does it work for you?

Reply:
Seems like the [CI had the same problem](https://github.com/rbaron/b-parasite/pull/13/checks?check_run_id=3498473009):

```bash
+ export SDK_ROOT=/opt/nRF5_SDK
+ SDK_ROOT=/opt/nRF5_SDK
+ export GNU_INSTALL_ROOT=/usr/bin/
+ GNU_INSTALL_ROOT=/usr/bin/
+ cd /github/workspace/code/b-parasite
+ make clean
rm -rf _build
+ make lint
find src -iname *.h -o -iname *.c | xargs clang-format -n -Werror
+ make
make: Nothing to be done for 'default'.
```

Reply:
You're (both :D) right, it had to do with the whitespace after the equals-sign. It works now after I've removed it and added a comment explaining that.